test/test-utils.ts

```diff
@@ -95,8 +95,14 @@ export const buildPPOMController = (args?: any) => {
     messenger: controllerMessenger.getRestricted({
       name: 'PPOMController',
     }),
-    refreshInterval: REFRESH_TIME_DURATION,
     ...args,
   });
   return ppomController;
 };
+
+// eslint-disable-next-line jsdoc/require-jsdoc
+export async function flushPromises() {
+  // Wait for promises running in the non-async timer callback to complete.
+  // From https://github.com/facebook/jest/issues/2157#issuecomment-897935688
+  return new Promise(jest.requireActual('timers').setImmediate);
```

Comment:
Do we need the `requireActual` here? In the past I've just used `new Promise(resolve => setImmediate(resolve))`?

Reply:
`new Promise(resolve => setImmediate(resolve))` is not working. May be different is about jest version.